### DISCUSSION TITLE
Data Visualisation section of the playbook

### DISCUSSION THREAD

**Discussion by Helen326** [nhsbsa/nhsbsa-digital-playbook#245] [Data]
Hi all, we have a number of Data Visualisation standards available and we are currently looking into uploading a Data Visualisation section to the playbook (which will in time sit alongside other Data standards).

**Comment by nhsbsa-sthil** [comment]
Hi @Helen326, we have a 'Data' category - is it ok if I move this discussion into there?

https://github.com/nhsbsa/nhsbsa-digital-playbook/discussions/categories/data

Thanks! 😄
